src/lib/Microsoft.Health.Fhir.R4.Ingest/Template/CodeValueFhirTemplateProcessor.cs

```diff
@@ -81,6 +86,12 @@ protected override Observation MergeObservationImpl(CodeValueFhirTemplate templa
 
             existingObservation.Status = ObservationStatus.Amended;
 
+            existingObservation.Category = null;

```

Comment:
Do we want to eliminate the categories on the existing observation? Are we assuming that the only ones there are the ones from our template?

Reply:
It's up for debate, but my preference is to overwrite. 

To me it seems strange to have one system sending in a category for an observation and then have another system send in another category for the same observation. I would assume that both systems would be consistent and send the same category for the same observation. 

I don't mind pulling out this particular line of code though. If no category is specified in the template then this line wipes out the category if it previously existed in the observation, which is a bit controversial. 

If we remove this line then we only overwrite if there is a category specified in the template. However, as a minor side effect, there would be no way to delete a category.

Thoughts?

Reply:
I am fine keeping it.  Wiping out is fine since if that is the template change there won't be a category for observations going forward.

Reply:
So just to make sure the only way this changes is template changes, so overwrite going forward is OK with me and we would have to assume some consistency due diligence on the template managers part.  Also if it runs through a profile on the way in that could also help with enforcement/consistency.

Reply:
@sordahl-ga through the connector/tool that is correct.  Changing the template is the only way to make modifications.  If an organization allowed observations created by the connector to be edited by others there could be an issue but our position is the connector should own the observations it creates and they aren't mutable by other actors.

Reply:
@dustinburson I was thinking of a use case (and maybe this is an incorrect usage of category) where a system would analyze observations and flag them for some reason (possibly for further investigation, or it's out of the norm or some other reason). Our system would then wipe that out if another reading in that time window comes in.

Reply:
@LiquidPT The scenario makes sense but I don't think that is an appropriate use of category according to the FHIR spec.  More appropriate would be using another observation that references the device one with an interpretation or a risk assessment.  If the underlying observation needs to be modified for the purpose you describe using the interpretation property (https://www.hl7.org/fhir/observation-definitions.html#Observation.interpretation) or an extension seems more appropriate.